frontend/public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
```

Comment:
We might want to change these two names?

Reply:
Sure, did you have any ideas in mind?

Reply:
What do you think about `Switchboard` and `Switchboard Application`.